docs/competition/team-support/battery-charging.md

```diff
@@ -7,44 +7,76 @@ original:
+### Equipment
+
+There are two locations involved in battery charging:
+
+- Helpdesk
+    - 1 x Discharged Battery Box (18L RUB), clearly labelled
+    - 1 x Charged Battery Box (18L RUB), clearly labelled
+    - 1 x LiPo Bag for damaged batteries, clearly labelled, red taped
+    - At least one roll of red insulating tape
+- Battery Charging Station
+    - At least `n` battery chargers
+    - Enough LiPo charging bags for every charger.
+    - 1 x Discharged Battery Box (18L RUB), clearly labelled
+    - 1 x Charged Battery Box (18L RUB), clearly labelled
+    - 1 x Damaged battery box (18L RUB), clearly labelled, red taped
+    - At least one roll of red insulating tape
+
+
+### Safe Use of Battery Charging Equipment - Battery Charging Station 
+
+- There should be `0.5A` of mains electricity available per charger.
+- It should be ensured that any power distribution / 4 ways is setup properly and able to handle the current draw.
+- Additionally, there should be no trailing wires across the floor to prevent charging batteries falling on the floor.
 
 ## Volunteer Requirements
 
-Every volunteer performing the Battery Charging role must have read this document fully. Any doubts must be discussed with Rich Barlow ASAP.
+Every volunteer performing the Battery Charging role must have read this document fully. Any doubts must be raised before beginning the role.
 
 ## Safety
 
 The biggest risk of damage or injury is from the banana plugs coming unplugged from the charger. These are the red and black plugs that go into the side of the charger. All of the chargers should have the cables cable-tied to the charger to make it difficult to remove these plugs. Under no circumstances should these red and black plugs be unplugged from the chargers.
 
 The batteries themselves have many built in safety features and, unless severely abused, are completely safe. All batteries undergoing charging must be placed inside a fireproof charging bag. In the extremely unlikely event of a fire due to a battery DO NOT attempt to tackle the fire - the batteries contain a flammable metal (Lithium) which cannot be safely tackled with most fire extinguishers.
 
-If at any point you have any doubts about safety, immediately summon the Competition Team Support Coordinator (Rich Barlow) and the Health & Safety Coordinator (Scarzy) via [Helpdesk](/competition/team-support/helpdesk).
+If at any point you have any doubts about safety, immediately ask the helpdesk to summon the Health & Safety officer.
 
 ## Procedures
 
-### Swapping a team's battery
+### Swapping a team's battery - Helpdesk
+
+- Teams will bring their discharged battery to the Helpdesk
+- Discharged batteries should be placed in the discharged battery box
+    - If the battery is damaged, red tape should be placed on it and it should be placed in the damaged LiPo bag instead of the discharged battery box. 
+- A charged battery should be retrieved from the charged battery box and handed to the team.
```

Comment:
We seem to have lost the reasoning behind this. It's pretty important that the runbook records _why_ things are done the way that they are so that we know what's reasonable to change about the way it's done.

Reply:
What reasoning do you think is needed here? Isn't this the way that battery charging is normally conducted.

Reply:
This is indeed the way it's normally conducted, however the runbook is about more than just recording _what_ is done. It's important to record _why_ things are done a particular way too. 

Without an understanding of why a piece of documentation suggests a given thing, it can be easy to end up following the docs "because the docs say so" and then either getting stuck unable to change a process, or ending up changing a process and finding out the hard way that things were the way they were for a good reason. Recording the reason upfront avoids both.

Recording the "why" of things is a general principle we're trying to keep in the runbook, though as it's based on lots of existing docs not everything has that yet. It feels like it would be a shame for this particular bit to go backwards when it already has the reasoning recorded.